plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/labeling/helper/StyledTextCalculationHelper.xtend

```diff
@@ -145,50 +153,64 @@ class StyledTextCalculationHelper {
 
 	// produces e.g. getterName() : returnTypeName
 	def dispatch StyledString dispatchGetStyledText(N4GetterDeclaration n4GetterDeclaration) {
-		var styledText = getLabelProvider.getSuperStyledText(n4GetterDeclaration);
-		val definedGetter = n4GetterDeclaration.definedGetter;
+		if (n4GetterDeclaration.definedGetter!==null)
+			return dispatchGetStyledText(n4GetterDeclaration.definedGetter);
+	
+		return getLabelProvider.getSuperStyledText(n4GetterDeclaration);
```

Comment:
rename `getLabelProvider` either to explicit function call`getLabelProvider()` or Xtend's shortcut`labelProvider` for better readability?